addons/website_livechat/models/discuss_channel.py

```diff
@@ -51,9 +51,6 @@ def _to_store_defaults(self, target):
             ),
         ]
 
-    def _get_visitor_history(self, visitor):
-        return visitor._get_visitor_history()
-
```

Comment:
The usage of this `_get_visitor_history` has been removed [here](https://github.com/odoo/odoo/pull/201472/files#diff-9e89ad87ed807955ee59189b6846578cb5b1f50ff9f0c6af41dc4f692f61363dL52).